backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -300,10 +300,32 @@ def get_volume(self, data):
         bookseries_volume = get_value(data, "book_series.volume[0]")
         return publication_volume or bookseries_volume
 
+    @staticmethod
+    def get_partial_publication_info(data):
+        publication_info = get_value(data, "publication_info", [])
+        only_publications = [
+            entry
+            for entry in publication_info
+            if entry.get("material", "publication") == "publication"
+            and (entry.get("artid") or entry.get("page_start"))
+        ]
+
+        if not only_publications:
+            return {}
+
+        return sorted(
+            only_publications, key=lambda item: item.get("year", float("inf"))
+        )[0]
+
     def get_pages(self, data):
```

Comment:
where is the test for this bugfix? did you try it out manually? how do you know it works?